src/Module/Manifest.php

```diff
@@ -36,12 +36,15 @@ public static function rawContent(array $parameters = [])
 		$theme = DI::config()->get('system', 'theme');
 
 		$manifest = [
-			'name'        => $config->get('config', 'sitename', 'Friendica'),
-			'start_url'   => DI::baseUrl()->get(),
-			'display'     => 'standalone',
-			'description' => $config->get('config', 'info', DI::l10n()->t('A Decentralized Social Network')),
-			'short_name'  => 'Friendica',
-			'icons'       => [
+			'name'			=> $config->get('config', 'sitename', 'Friendica'),
```

Comment:
Ah-ha! Please don't use tabulations to align columns as the alignment (or lack thereof) then depends on the tabulation size.

Reply:
RIP. Thanks I'll change them all!!! :joy: